src/inmanta/server/services/orchestrationservice.py

```diff
@@ -675,7 +675,10 @@ async def _check_version_info(
             module_version = module_data.version
             if (inmanta_module_name, module_version) not in base_version_data:
                 raise BadRequest(
-                    "Cannot perform partial export because of version mismatch for module %s." % inmanta_module_name
+                    "Cannot perform partial export because the source code for module %s in this partial version is different "
+                    "from the source code of the base version. Consider running a full export instead. Alternatively, if you "
```

Comment:
```suggestion
                    "from the currently registered source code. Consider running a full export instead. Alternatively, if you "
```

I'm not sure about this. But I feel like "base version" is a bit too detailed, and it's not a term we expose elsewhere. As far as the user is concerned, I think the abstract description in my suggestion says all?